src/components/ComponentRelation/cr-modals.tsx

```diff
@@ -53,7 +53,7 @@ export const DefineComponentRelationModal: React.FC<DefineComponentRelationModal
       description={
         <>
           Nudging references another component by digest.{' '}
-          <ExternalLink href="https://konflux-ci.dev/docs/building/component-nudges/">
+          <ExternalLink href="https://konflux-ci.dev/docs/building/component-nudges/#customizing-nudging-prs">
```

Comment:
put this into `src/consts/documentation.tsx`

Reply:
done! :)